### DISCUSSION TITLE
请问能不能切换菜单的时候，不自动修改浏览器标签的标题？

### DISCUSSION THREAD

**Discussion by ShadowJobs** [ant-design/ant-design#37355] [Q&A] [answered]
这个设计太坑了，老板现在让我把标题固定住，我都不知道怎么跟他解释。他问为什么别的网站都能固定住。
我尝试写在document.ejs，defaultsettings的title，都不管用
我尝试在app.tsx里加title，结果，加倒是加上 ，但是加在了原来的菜单内容的后面T_T

**Comment by yoyo837** [comment, marked as answer]
这和antd有什么关系，antd不参与设置任何title

**Reply by ShadowJobs** [reply]
抱歉，我去pro那儿提
